telegram/files/sticker.py

```diff
@@ -72,5 +83,67 @@ def de_json(data, bot):
         data = super(Sticker, Sticker).de_json(data, bot)
 
         data['thumb'] = PhotoSize.de_json(data.get('thumb'), bot)
+        data['mask_position'] = MaskPosition.de_json(data.get('mask_position'), bot)
 
         return Sticker(**data)
+
+    @staticmethod
+    def de_list(data, bot):
+        if not data:
+            return list()
+
+        stickers = list()
+        for sticker in data:
+            stickers.append(Sticker.de_json(sticker, bot))
+
+        return stickers
+
+
+class StickerSet(TelegramObject):
+    def __init__(self, name, title, contains_masks, stickers, **kwargs):
+        # TODO: telegrams docs claim contains_masks is called is_masks
+        # remove these lines or change once we get answer from support
+        self.name = name
+        self.title = title
+        self.contains_masks = contains_masks
+        self.stickers = stickers
+
+        self._id_attrs = (self.name,)
+
+    @staticmethod
+    def de_json(data, bot):
+        if not data:
```

Comment:
Probably we need to raise an exception here or make it return empty stickerset. Exception looks better in my opinion